packages/fiori/src/TimelineItem.ts

```diff
@@ -89,8 +89,14 @@ class TimelineItem extends UI5Element implements ITimelineItem {
 	@property()
 	subtitleText?: string;
 
+	@property({ type: Boolean })
+	_firstItemInTimeline = false;
+
+	@property({ type: Boolean })
+	_isNextItemGroup = false;
+
 	@property({ noAttribute: true })
-	forcedTabIndex?: string;
+	forcedTabIndex?: string = "-1";
```

Comment:
you can't have value with ? and with initializer. ? means that the property is optional and initially it won't have value.